ng-dev/pr/merge/cli.ts

```diff
@@ -56,7 +55,7 @@ async function handler({
   dryRun,
   ignorePendingReviews,
 }: Arguments<MergeCommandOptions>) {
-  await mergePullRequest(pr, {branchPrompt, forceManualBranches, dryRun, ignorePendingReviews});
+  await mergePullRequest(parsePrNumber(pr), {branchPrompt, forceManualBranches, dryRun, ignorePendingReviews});
```

Comment:
Rather than parsing here, you should be able to leverage the `coerce` value in the `positional` definition: https://yargs.js.org/docs/#api-reference-coercekey-fn

That way you by the time its done parsing, it is already coerced into a number.

Reply:
Ahhh, this is what you meant. OK. Updated.